src/init.sh

```diff
@@ -170,3 +170,33 @@ function init_help()
     '  init --remote <user>@<ip>:<port> - Set remote fields in the kworkflow.config file.' \
     '  init --target <target> Set the default_deploy_target field in the kworkflow.config file'
 }
+
+# This function creates a new ssh key.
+#
+# @ssh_dir  : directory where the key will be stored.
+# @key_name : this is the name of the key, usually
+#             used to avoid overwriting keys.
+#             If not given, defaults to "kw_ssh_key".
+#
+# Returns:
+# ENOTDIR                   : a file named @ssh_dir already exists and
+#                             is not a directory.
+# EEXISTS                   : key already exists.
+# ssh-keygen's return value : returns ssh-keygen's value if @ssh_dir is valid.
+function create_ssh_key()
```

Comment:
I think it makes more sense to move this function to `kw_ssh` and includes it in this file when you need it.

Reply:
This function should appear before the parse and help functions.